setup.py

```diff
@@ -45,9 +46,12 @@
     "tests": tests_require,
 }
 
-extras_require["all"] = []
-for reqs in extras_require.values():
-    extras_require["all"].extend(reqs)
+all_requires = []
```

Comment:
doubt: why do we need to skip elasticsearch extras? Is this the reason some were "forcefully" being installed (always 6 and then move to 7, if 7 was specified?)

Reply:
This took me a while to discover... When we run `pip install -e .[all, elasticsearch7]` as before, because `all` extra lists `elasticsearch6` as a dependency, it would get installed and override the installation of elasticsearch7 (or vice-versa - no install guarantees because both are dependencies). All other packages (e.g. https://github.com/inveniosoftware/invenio-records-rest/blob/master/setup.py#L68) that use the elasticsearch extras, go through this filtering process to prevent this from happening.